app/talk/init.cjsx

```diff
@@ -49,13 +52,46 @@ module.exports = React.createClass
       @setBoards({}, nextProps)
 
   setBoards: (propValue, props = @props) ->
+    author_ids = []
     talkClient.type('boards').get(section: props.section, page_size: 20, page: props.location.query.page)
       .then (boards) =>
         boardsMeta = boards[0]?.getMeta()
         @setState {boards, boardsMeta, loading: false}
+        boards.map (board) ->
+          author_ids.push board.latest_discussion.user_id
```

Comment:
I think this is the original author of the discussion, rather than the author of the latest comment (which is what we really want to display).